src/content/docs/cypher/import_export_db.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EXPORT/IMPORT DATABASE
+---
+
+Kùzu allows you to import/export database using query statements. 
```

Comment:
Can you add a line here at the start why you may want to import/export a database? (Something along the lines of if you need to migrate the contents of an existing database to a different version of Kùzu). Also, maybe we should mention the cases where the import/export may not work (e.g., backward compatibility is not guaranteed).